host/tests/configs/example_rack_config.yml

```diff
@@ -0,0 +1,113 @@
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2023 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+# This config file creates your environment, defaults are setup internally unless otherwise overridden.
+# you will need at least 1 rack and 1 slot to operate.
+# each module e.g. console options, will be off by default unless otherwise stated
+
+# Depreciated
+# slotx: address:  IP of device while running locally, replaced with slotx: ip
+# slotx: deviceConsole:  IP of device while running locally, replaced with slotx: devices
+
+# Data that is global to all tests.
+globalConfig:
+    includes:
+        # [ includes: ]
+        # [ deviceConfig: "required.yml file"  ]
+        deviceConfig: "example_device_config.yml"
```

Comment:
For this to work with the device config you've added in the configs' folder, this needs changing to deviceConfig.yml. Not sure how important this is.

Reply:
not very sreenivas really just wanted an example, I add a link to where it comes from I guess that would be useful.